lighthouse-cli/test/fixtures/static-server.js

```diff
@@ -79,6 +90,8 @@ function requestHandler(request, response) {
       headers['Content-Type'] = 'image/webp';
     } else if (filePath.endsWith('.json')) {
       headers['Content-Type'] = 'application/json';
+    } else if (filePath.endsWith('.html') || filePath === '/') {
+      headers['Content-Type'] = 'text/html';
```

Comment:
this feels 100% unrelated to the index part, right? 

why cause conflicts for yourself in https://github.com/GoogleChrome/lighthouse/pull/9542 :)